elasticsearch-jmeter/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,793 @@
+        "kibanaTemplates": {
+            "yes": "kibana-resources.json",
+            "no": "empty-resources.json"
+        },
+        "kibanaTemplateFileName": "[variables('kibanaTemplates')[variables('kibana')]]",
+        "ubuntuScripts": [
+            "[concat(variables('scriptsLocation'), 'elasticsearch-ubuntu-install.sh', variables('_artifactsLocationSasToken'))]",
+            "[concat(variables('scriptsLocation'), 'kibana-install.sh', variables('_artifactsLocationSasToken'))]",
+            "[concat(variables('sharedScriptsLocation'), 'vm-disk-utils-0.1.sh')]",
+            "[concat(variables('sharedScriptsLocation'), 'afs-utils-0.1.sh')]"
+        ],
+        "centosScripts": [
+            "[concat(variables('scriptsLocation'), 'elasticsearch-centos-install.sh', variables('_artifactsLocationSasToken'))]",
+            "[concat(variables('scriptsLocation'), 'kibana-centos-install.sh', variables('_artifactsLocationSasToken'))]",
+            "[concat(variables('sharedScriptsLocation'), 'vm-disk-utils-0.1.sh')]"
+        ],
+        "windowsScripts": [
+            "[concat(variables('scriptsLocation'), 'elasticsearch-windows-install.ps1', variables('_artifactsLocationSasToken'))]"
+        ],
+        "authSet": {
+            "password": "[variables('adminPassword')]",
+            "sshKey": "[variables('sshPublicKey')]",
+            "type": "[variables('authenticationType')]"
+        },
+        "configurationObject": {
+            "password": {},
+            "sshPublicKey": {
+                "disablePasswordAuthentication": "true",
+                "ssh": {
+                    "publicKeys": [{
+                        "path": "[concat('/home/', variables('adminUsername'), '/.ssh/authorized_keys')]",
+                        "keyData": "[variables('sshPublicKey')]"
+                    }]
+                }
+            }
+        },
+        "ubuntuSettings": {
+            "imageReference": {
+                "publisher": "Canonical",
+                "offer": "UbuntuServer",
+                "sku": "14.04.4-LTS",
+                "version": "latest"
+            },
+            "linuxConfiguration": "[variables('configurationObject')[variables('authenticationType')]]",
+            "managementPort": "22",
+            "extensionSettings": {
+                "marvel": {
+                    "publisher": "Microsoft.Azure.Extensions",
+                    "type": "CustomScript",
+                    "autoUpgradeMinorVersion": true,
+                    "typeHandlerVersion": "2.0",
+                    "settings": {
+                        "fileUris": "[variables('ubuntuScripts')]",
+                        "commandToExecute": "[concat('bash elasticsearch-ubuntu-install.sh -wn ', variables('esSettings').marvelClusterName, ' -v ', variables('esSettings').version, ' -d ', variables('esSettings').marvelHosts, ' -k ')]"
+                    }
+                },
+                "master": {

```

Comment:
these vars seem to differ only in the cmd to execute, let's reduce the var to just the part that varies.